README.md

```diff
@@ -135,6 +135,27 @@ go get github.com/Azure/go-autorest/autorest/date
 go get github.com/Azure/go-autorest/autorest/to
 ```
 
+### Using with GO Modules
+Since [v12.0.1](https://github.com/Azure/go-autorest/pull/386), this repository added support for Go modules on per package basis. If your code directly and/or indirectly depends on previous versions of autorest library, use replacements to tell Go to use the latest version of the modules from this repo:
+
+```
+module your_module_here
+
+go 1.12
+
+require (
+	github.com/Azure/azure-sdk-for-go v31.1.0+incompatible
+	github.com/Azure/go-autorest/autorest v0.5.0
+	...
+)
+
+replace (
+	github.com/Azure/go-autorest => github.com/Azure/go-autorest/autorest v0.5.0
```

Comment:
This solution relies on vendoring which is IMO sub-optimal as not everybody vendors their dependencies.  Instead, replace `go-autorest` with whatever version is specified in azure-sdk-for-go's `Gopkg.toml`.
`github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible`
Works for vendoring/vendorless builds (tested on the sample you provided).

Reply:
Updated.

Reply:
@jhendrixMSFT , don't mean rush you, but do this pr will be merged? If not, I would like to close the pr. Just trying to clean up my queue.